oneview_redfish_toolkit/strategy_multiple_oneview.py

```diff
@@ -101,3 +109,22 @@ def _run_action(resource_id, resource_get, function_get, resource,
     return multiple_oneview.query_ov_client_by_resource(resource_id,
                                                         resource, function,
                                                         *args, **kwargs)
+
+
+def _get_resource_id_by_filter(kwargs, resource_uri):
```

Comment:
Maybe "filter_parameter", "filter_property" or "property_name" would fit better for this argument.